.happy/terraform/modules/schema_migration/main.tf

```diff
@@ -227,6 +227,11 @@ resource aws_sfn_state_machine sfn_schema_migration {
           "ErrorEquals": ["States.TaskFailed"],
           "IntervalSeconds": 30,
           "MaxAttempts": 1
+      }, {
+          "ErrorEquals": ["AWS.Batch.TooManyRequestsException", "Batch.BatchException", "Batch.AWSBatchException"],
```

Comment:
These should be above `States.TaskFailed` since the first match error used.

> When one of these states reports an error and there's a Retry field, Step Functions scans through the retriers in the order listed in the array. When the error name appears in the value of a retrier's ErrorEquals field, the state machine makes retry attempts as defined in the Retry field.

Reply:
it is, it looks like you're commenting on an outdated version of the file!